packages/graphiql/cypress/integration/init.spec.js

```diff
@@ -0,0 +1,53 @@
+
+const testQuery = `{
+longDescriptionType {
+  id
+  image
+  hasArgs
+  test {
+    id
+    isTest
+    __typename
+  }
+ }
+}`
+
+const mockSuccess = {
+  "data": {
+    "longDescriptionType": {
+      "id": "abc123",
+      "image": "/images/logo.svg",
+      "hasArgs": "{\"defaultValue\":\"test default value\"}",
+      "test": {
+        "id": "abc123",
+        "isTest": true,
+        "__typename": "Test"
+      }
+    }
+  }
+}
+
+
+describe('GraphiQL On Initialization', function() {
+  it('Renders without error', function() {
+    cy.visit(`/?query=${testQuery}`)
+  })
+
+   it('Executes a query that has the expected resut', function() {
+    cy.get('.execute-button').click()
+    cy.window().then((w) => {
+      cy.expect(JSON.parse(w.g.resultComponent.viewer.getValue())).to.deep.equal(mockSuccess)
+    })
+  })
+
+  it('Toggles doc pane off', function() {
+    cy.get('.docExplorerWrap button.docExplorerHide').should('be.visible').click({ multiple: true })
```

Comment:
The 'be.visible' assertion is [unnecessary](https://docs.cypress.io/guides/core-concepts/interacting-with-elements.html#Actionability).

```suggestion
    cy.get('.docExplorerWrap button.docExplorerHide').click({ multiple: true })
```

What does the `multiple: true` do? I've never used it. Docs say "Serially click multiple elements" but that doesn't seem relevant here?